src/app/components/ImageCard/ImageCard.module.scss

```diff
@@ -0,0 +1,22 @@
+.image-card {
```

Comment:
In some of the other modules we were naming things like `imageCard` so that we could reference them in the .tsx file like `styles.imageCard` rather than `styles['image-card']`. 

I don't have a strong preference either way. Thought I'd mention it in case you had any opinions on it one way or another.